src/app/components/PropertyMap.tsx

```diff
@@ -328,17 +328,41 @@ const PropertyMap: FC<PropertyMapProps> = ({
             closeOnClick={false}
             onClose={() => setPopupInfo(null)}
           >
-            <div>
-              <p className="font-semibold text-md p-1">
-                {popupInfo.feature.address}
-              </p>
+            <div
+              style={{
+                display: 'flex',
+                justifyContent: 'space-between',
+                flexWrap: 'nowrap',
+              }}
```

Comment:
just a nitpick, but updating this to tailwind to keep it consistent with the codebase would be nice

Reply:
All good. Done.